lib/MusicBrainz/Server/Data/CoverArtArchive.pm

```diff
@@ -234,21 +234,6 @@ sub exists_for_release_gid {
     );
 }
 
-sub mime_types {
```

Comment:
I'll remove that commit because I'm not sure I follow the logic in the commit message anymore (it seems to have been authored years ago, before we removed other cover art providers).